lib/server/ServerRenderer.js

```diff
@@ -72,24 +71,33 @@ function makeClientAppStartScript(clientAppRequirePath, environmentConfig, boots
         "});";
 }
 
-function makeBootstrappingLoader(scriptBody, clientAppUrl, isAsync) {
-    var loadAsync = isAsync ? "s.async = true;\n" : "";
```

Comment:
this is not needed, scripts that are dynamically injected are done async by default, we would need this attribute to make the load blocking. So this line is really doing nothing but confuse the user